Nodejs/Product/TestAdapter/TestFrameworks/TestFramework.cs

```diff
@@ -111,6 +111,11 @@ class TestFramework {
             return path;
         }
 
+        private string WrapTestNameWithQuot(string path)
```

Comment:
Lastly including this near the WrapWithQuotes method will make the code a tad more organized